src/logic/operate.js

```diff
@@ -49,9 +49,6 @@ const operate = (numberOne, numberTwo, operation) => {
   }
 
   if (operation === '÷') {
-    if (numberOne === '0' && numberTwo === '0') {
-      return 'Math Error';
-    }
     if (numberOne !== '0' && numberTwo === '0') {
       return 'Math Error';
     }
```

Comment:
This is all you need
```
if (operation === '÷') {
    if (numberTwo === '0') {	
      return 'Math Error';	
    }	
   return a.div(b).valueOf();
}
```

Reply:
omg, didn't see that, thanks for the feedback, working on it now. 

Kind regards,
bafiam